src/Appwrite/Event/Realtime.php

```diff
@@ -0,0 +1,228 @@
+<?php
```

Comment:
Big parts here might be a good fit for the Messaging library.

Reply:
Got this class removed 👏🏻